src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1844,6 +1844,20 @@ To provide the _firstName_ and _lastName_ variables parameters to the submitted
 
 The `SubmitJobNoWait` or `SubmitJobAndWait` templates behavior is different in the sense that the latter will wait until the submitted workflow terminates.
 
+In case an error occurs in the chained jobs, a user may want to escalate this error from the submitted (child) job to the parent job by throwing an exception. It is possible to integrate this behavior in the `SubmitJobAndWait` implementation using the <<_scheduler_api,*schedulerapi*>> binding as described in the code below.
+[source,groovy]
+----
+jobid = variables.get("jobID")
+jobState = schedulerapi.getJobState(jobid)
+jobStatus = jobState.getStatus().toString()
+isJobSuccessful = !(jobStatus.equals("FAILED") || jobStatus.equals("CANCELED") || jobStatus.equals("KILLED") || (jobStatus.equals("FINISHED") && jobStatus.getNumberOfFaultyTasks() > 0))
```

Comment:
you need to fix this test :-)

Reply:
!(jobStatus.equals("FAILED") && !jobStatus.equals("CANCELED") && ...

Reply:
I applied a fix and double checked this test with real examples.